frontend/src/scenes/data-management/events/DefinitionHeader.tsx

```diff
@@ -164,13 +173,10 @@ function RawDefinitionHeader({
 
 export function EventDefinitionHeader({
```

Comment:
Not strictly related to this PR, but "Last modified"'s layout gets weird at times:
<img width="350" alt="Screen Shot 2022-03-11 at 15 15 18" src="https://user-images.githubusercontent.com/4550621/157885455-a512fab7-c05a-411f-8242-d613eb65920a.png">